lib/fastlane/plugin/wpmreleasetoolkit/actions/ios/ios_get_store_app_sizes.rb

```diff
@@ -111,7 +115,7 @@ def self.authors
         end
   
         def self.is_supported?(platform)
-          platform == :ios
+          [:ios, :mac].include?(platform)
```

Comment:
Totally out of topic question. But it's a bit confusing that this code which supports iOS and macOS lives under `ios/`.

This obviously is not a critique, since there is not `macos` or `apple` folder in the library structure. Just pointing it out for consideration.

I can live with code from a folder named "iOS" running for macOS, too, no dramas 😄 I guess it just stood out because of how small the diff is.